dyno/panda/dyno_panel.py

```diff
@@ -0,0 +1,133 @@
+import tkinter as tk
+from tkinter import font
+from PIL import Image, ImageTk
+import serial
+import time
+
+#Define constants
+NUMSENSORS1 = 2 #RPM AND TEMPERATURE
+NUMSENSORS2 = 2 #JOULEMETER
+#---------------------
+
+#Images
+dino_path = "./img/dino.gif"
+start_path = "./img/start.png"
+#---------------------
+
+#Colors
+color_bg = "#16495C"
+color_fg = "#DA8669"
+color_label = "#B1D9DB"
+#---------------------
+
+#Placement
+LABEL_COL1_PLACE = {'relx':0.15, 'rely':0.6}
+LABEL_COL2_PLACE = {'relx':0.55, 'rely':0.6}
+LABEL_SPACE = 0.08
+#---------------------
+
+#Open the serial port that the MCU is connected to
+'''ser = serial.Serial('/dev/ttyUSB0', 9600, timeout=0.5)'''
+#---------------------
+
+#Update the variables for the labels
+def update_labels():
+    #Read in input from MCU sensors
+    '''for x in range (0, NUMSENSORS1): #read sensor statements
+        val = ser1.readline().decode('ascii')
+        prefix = val[0:3]
+        my_dict[prefix] = float(val[5:-2])
+    for x in range (0, NUMSENSORS2): #read JOULEMETER statements
+        val = ser2.readline().decode('ascii')
+        prefix = val[0:3]
+        my_dict[prefix] = float(val[5:-2])'''
+    for key in my_dict.keys():
+        my_dict[key][1] = my_dict[key][1] + 1
+    my_list.append(str(my_dict))
+
+    temp = list(my_dict.values())
+    
+    for i in range(num_vars):
+        var_array[i].set(temp[i][0] + str(temp[i][1]) + temp[i][2])
+
+    window.after(100, update_labels)
+#---------------------
+
+#Define on_click function for button
+def handle_click(event):
+    if button["text"] == "Pause":
+        button["text"] = "Run"
+        #ser.write('P;')
```

Comment:
Yeah having commented code out in many places like this scares me a little. Ran into quite a few issues with this when writing motor control code.